Admin/AdminHelper.php

```diff
@@ -335,6 +334,8 @@ private function pathExists(PropertyAccessorInterface $propertyAccessor, $entity
             return true;
         } catch (NoSuchPropertyException $e) {
             return false;
+        } catch (UnexpectedTypeException $e) {
```

Comment:
> This is only for sf2.3.

In this case, could you add a comment saying since which SF version we can remove this and why? Thanks.